osm2streets/src/transform/intersection_geometry.rs

```diff
@@ -0,0 +1,124 @@
+use abstutil::Timer;
+use geom::{Circle, Distance};
+
+use crate::{ControlType, StreetNetwork};
+
+pub fn generate(streets: &mut StreetNetwork, timer: &mut Timer) {
```

Comment:
git didn't detect this as a move from `initial.rs`, but the logic comes from there